utils/io/fileutils/files.go

```diff
@@ -16,7 +17,11 @@ import (
 	"strings"
 )
 
-const SYMLINK_FILE_CONTENT = ""
+const (
+	SYMLINK_FILE_CONTENT                  = ""
+	File                 TraverseItemType = "file"
+	Folder               TraverseItemType = "dir"
```

Comment:
I suggest using *Dir* instead of *Folder*.

Reply:
Changed.